TERITORI/teritori-dapp
# Settings page [:shipit: ]

- implement setting page for desktop & mobile responsive

It needs a new ENV KEY:
`NFTStorageAPI` with teritori pinata API KEY.

How to test:

Go to the top right menu: click on it, then click on settings or https://deploy-preview-472--testitori.netlify.app/settings

Please note that the "user profile customization" part of this task was changed to navigate to the TNS manage page.
I left the files ( committed to the repo UserProfileModal.tsx) as by the previous developer.

Changes ACK'ed by: @zxxma
